.goreleaser.yml

```diff
@@ -205,3 +205,21 @@ docker_manifests:
     image_templates: [*amd64-image, *arm64-image]
   - name_template: "{{ .Env.REGISTRY }}/{{ .Env.ORG }}/{{ .Env.REPO }}:{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
     image_templates: [*amd64-image, *arm64-image]
+
+sboms:
```

Comment:
Is there a way to use env vars to conditionally select the build targets and enable/disable sboms? It would be nice to maintain just a single `.gorleaser.yml` file.

Reply:
The only way I know is to build all targets, which is very time-consuming, especially building Windows. We only need Unix targets to build the container images of the snapshots used by the tests.

As we need to do a snapshot release to create the docker images we cannot use 
```
goreleaser build --id stringArray
```

that builds only the specified build IDs but does not create the container images.

Since `goreleaser release` only supports passing in a config file, that is why it is setup using a special trimmed down config which only constructs the container image to run the tests.